hydra_agent/collections_endpoint.py

```diff
@@ -201,8 +224,38 @@ def load_from_server(
                 redis_connection.delete(key)
         # save the new data.
         self.redis_graph.commit()
-#        for node in self.redis_graph.nodes.values():
-#            print("\n",node.alias)
+        # for node in self.redis_graph.nodes.values():
+        # print("\n",node.alias)
+
+    def insert_server(
```

Comment:
insert_server is not very descriptive please use better function names. Also, update the doctrings

Reply:
I fixed all the other comments on my local repo. Thanks for the notes @xadahiya 

Regarding the function names, I tried to follow the previous standard. But guess I'll refactor them with the new design we discussed on the call